src/commands/autocomplete.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import Command, {flags} from 'cli-engine-command'
+import Plugins from '../plugins'
+import path from 'path'
+import {CustomColors} from 'cli-engine-command/lib/output'
+
+export default class Autocomplete extends Command {
+  static topic = 'autocomplete'
+  static description = 'autocomplete installation instructions'
+  static args = [
+    {name: 'shell', description: 'shell to use', required: false}
+  ]
+  static flags = {
+    location: flags.boolean({hidden: true}),
+    instructions: flags.boolean({description: 'display install instructions for the shell'})
+  }
+
+  async run () {
+    if (this.config.windows) {
+      this.out.warn('Autocomplete is not currently supported in Windows')
+      return
+    }
+
+    const autocompletePath = path.join(this.config.dataDir, 'client', 'autocomplete')
```

Comment:
@dickeyxxx I think I going to error here on brand new v6 installs b/c of the cli vs client dir thing we talked about earlier